src/components/board.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import sinon from 'sinon'
+import Board from './board';
+
+const BOARD_ROW = '.board-row';
+const CLICK = 'click';
+const GAME_TILE = '.game-tile';
+
+test('a 3x3 board has 3 rows', () => {
+  const tiles = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const board = shallow(<Board tiles={tiles} />);
+  expect(board.find(BOARD_ROW).length).toBe(3);
+});
+
+test('a 3x3 board row has 3 tiles and 9 tiles in total', () => {
+  const tiles = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const board = mount(<Board tiles={tiles} />);
+  const rows = board.find(BOARD_ROW);
+  expect(rows.first().find(GAME_TILE).length).toBe(3);
+  expect(rows.find(GAME_TILE).length).toBe(9);
+});
+
+test('an empty 3x3 board displays available position numbers', () => {
+  const tiles = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const board = mount(<Board tiles={tiles} />);
+  const allTiles = board.find(GAME_TILE);
+  allTiles.forEach(
+    (t, index) => {
+      expect(t.text()).toBe((tiles[index]).toString());
```

Comment:
Generally, I do not like putting assertions in for loops. Remember, part of the goal is to get good test output when a test fails. If this test fails, it might be difficult to determine specifically which tile had the failure.

That being said, I did not run this test and see the failure, so if the output message is clear, then it should be ok